addons/website/views/snippets/s_company_team.xml

```diff
@@ -2,20 +2,17 @@
 <odoo>
 
 <!--
-TODO review in master: this snippet's images use contenteditable="true" to be
-editable, their parent being o_not_editable/contenteditable="false" to prevent
-adding text by mistake. After the first save, contenteditable="true" are lost,
-so this is a bad practice / requires a better system to do this in the future.
-Meanwhile, this is fixed in JS (see COMPANY_TEAM_CONTENTEDITABLE).
+TODO in master: remove contenteditable="true" from <img> tags. Those were part
+of a stable fix to allow re-editing, superseded by .o_editable_media class
 -->
 <template id="s_company_team" name="Team">
-    <section class="s_company_team pt48 pb48">
+    <section class="s_company_team pt48 pb48" data-vxml="001">
```

Comment:
So, that should be enough in our case ? @dmo-odoo @qsm-odoo

Reply:
If you patch `_patchForComputeSnippetTemplates` as explained too, and you should test it:
- Create a 16.0 database without your branch, add a s_company_team snippet in a page
- Checkout your branch, then restart your server without updating any app
- Enter edit mode, click on the snippet you previously added -> it should show a message that says it is deprecated in the right panel (not the case if you don't patch `_patchForComputeSnippetTemplates` to add the data-xml attribute?)
- Add a new company team snippet -> the image should be properly editable (not the case if you don't patch `_patchForComputeSnippetTemplates` to add the `o_editable_media` classes?)
- Now update your apps and remove your JS patch temporarily -> test again both things, it should still work the same

=> This is how you should ensure that every part of your code is working as expected.

Reply:
Thanks for the detailed explanation ! ❤️

Reply:
Thank you indeed !
Just FYI, with the code as it is (without the patch you mention), after updating a database whose website has these snippets, the "deprecated message" correctly appears when selecting said snippets. I suppose the patch if for those that do not update the templates (not sure how the python code is updated without the templates being also updated when in production but I've heard that it happened indeed).